src/environments/environments/util.py

```diff
@@ -47,31 +47,10 @@ def process_odom(odom: Odometry):
         return [position.x, position.y, orientation.w, orientation.x, orientation.y, orientation.z, lin_vel.x,
                 ang_vel.z]
 
-def process_lidar(lidar: LaserScan):
```

Comment:
Don't think we should remove these functions. May be useful in the future - especially the process_lidar one.

Reply:
What is the purpose of process_lidar? The lidar doesn't actually send values for intensity does it? And then if we make the standard reduce lidar include the number of points, in theory you could just select every point and that would be the same as process_lidar?

Reply:
Actually yeah that's a good point. Im with you on that one now

Reply:
Fixed in latest commit